src/checkUsersTry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function checkUsersTry(usersTry, numberToGuess) {
+  if (isNaN(+usersTry) || usersTry.length !== 4) {
```

Comment:
You don't need to use the operator `+` to convert `usersTry` to a number because the function isNaN does it inside.